sklearn/gaussian_process/tests/test_gpr.py

```diff
@@ -652,3 +652,27 @@ def test_gpr_predict_error():
     err_msg = "At most one of return_std or return_cov can be requested."
     with pytest.raises(RuntimeError, match=err_msg):
         gpr.predict(X, return_cov=True, return_std=True)
+
+
+def test_y_std_with_multitarget_normalized():
+    """
+    Regression test for issues #17394 and #18065.
+    Check if GPR can compute y_std in predict() method when normalize_y==True
+    in multi-target regression.
+    """
+    X_train = np.random.rand((11, 10))
+    # 6 target features -> multi-target
```

Comment:
no need for this comment. Variable names and the code should be as much as possible self-explanatory.